catroid/src/main/java/org/catrobat/catroid/ui/controller/LookController.java

```diff
@@ -316,6 +323,7 @@ public void loadDroneVideoImageToProject(String defaultImageName, int imageId, A
 	private void copyImageToCatroid(String originalImagePath, Activity activity, List<LookData> lookDataList,
 			LookFragment fragment) {
 		try {
+			boolean lego = SettingsActivity.isMindstormsEV3SharedPreferenceEnabled(activity);
```

Comment:
Is it always a lego image if the settings are enabled?

Reply:
There is no other way (that I could figure out) without interfering with the user/changing the usability for the user.